src/test/codegen/unwind-extern-imports.rs

```diff
@@ -0,0 +1,41 @@
+// compile-flags: -C no-prepopulate-passes
+
+#![crate_type = "lib"]
+#![feature(unwind_attributes)]
+
+extern {
+// CHECK: Function Attrs: nounwind
```

Comment:
This line wants to look more like `Function Attrs:{{.*}}nounwind` to allow for other attributes to appear in this comment as well.